src/ansys/geometry/core/designer/body.py

```diff
@@ -1208,6 +1208,10 @@ def wrapper(self: "Body", *args, **kwargs):
 
         return wrapper
 
+    def _reset_tessellation_cache(self):  # noqa: N805
+        """Reset the cached tessellation for a body."""
+        self._template._tessellation = None
+
```

Comment:
Modify the wrapper function to also call this function in line 1206 - that way we make sure that we always use the same function to reset the tessellation.